joblib/test/test_pool.py

```diff
@@ -433,3 +436,46 @@ def test_workaround_against_bad_memmap_with_copied_buffers(tmpdir):
     finally:
         p.terminate()
         del p
+
+
+def test__strided_from_memmap(tmpdir):
+    fname = tmpdir.join('test.mmap').strpath
+    size = 5 * mmap.ALLOCATIONGRANULARITY
+    offset = mmap.ALLOCATIONGRANULARITY + 1
+    # This line creates the mmap file that is reused later
+    memmap_obj = np.memmap(fname, mode='w+', shape=size)
+    # filename, dtype, mode, offset, order, shape, strides, total_buffer_len
+    memmap_obj = _strided_from_memmap(fname, dtype='uint8', mode='r+',
+                                      offset=offset, order='C', shape=size,
```

Comment:
Turns out with `mode = 'r+'` you can change the size of the memmap file. With `mode = 'r'`, you get an error so I'll use `mode = 'r'` to avoid this kind of issues.